functions/stats.ts

```
@@ -0,0 +1,298 @@
+  return streakData;
+}
+
+
+/**
+ * Get the players data with the longest streak of days tagged
+ * @param players The array of players
+ * @returns Array of players data with longest streak of days tagged
+ */
+function getPlayersDataWithLongestStreakDaysTagged(
+  players: Player[]
+): Array<{}> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)